.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
```

Comment:
Its a shame precommit doesnt have an `extends` option. Discussion on that here with suggested hacks from the author of precommit included https://github.com/pre-commit/pre-commit/issues/450#issuecomment-405616722